dotcom-rendering/src/web/server/articleTemplate.ts

```diff
@@ -196,13 +196,7 @@ https://workforus.theguardian.com/careers/product-engineering/
 
                 <script type="module">
                     window.guardian.mustardCut = true;
-                    window.guardian.gaPath = "${gaPath.modern}";
-                </script>
-
-                <script nomodule>
-                    // Browser fails mustard check
-                    window.guardian.mustardCut = false;
```

Comment:
is this ok to remove?

Reply:
👍 We have `window.guardian.mustardCut` set to true above so I think it's fair to say we should also keep the code that sets it to false.

I think the potential use case here is another team depending on this property. I'm not aware of this being the case but we can't be sure and I don't think we lose anything but having this property available